### PR TITLE
Add note about manual confirmation to reservation info page (TTVA-92)

### PR DESCRIPTION
**Pull request by jopesy**
If the resource requires manual confirmation, show it in reservation details section in the reservation information view.

Screenshot 2023-05-23 at 14 08 20

Refs TTVA-92

### CODE REVIEW

File: app/pages/resource/reservation-info/ReservationInfo.js [hunk 1]
```diff
@@ -117,10 +117,20 @@ function renderReservationPeriodInfo(resource, t) {
   );
 }
 
+function renderRequiresManualConfirmationText(t) {
+  return (
+    <p className="needs-manual-confirmation-text">
+      <img alt="" className="app-ResourceHeader__info-icon" src={iconClock} />
```
Comment (danjacob-anders) [inline app/pages/resource/reservation-info/ReservationInfo.js]: Probably should be an alt text here for accessibility (or `aria-hidden`)

Comment (jopesy) [reply app/pages/resource/reservation-info/ReservationInfo.js]: Good point

Comment (jopesy) [reply app/pages/resource/reservation-info/ReservationInfo.js]: Added the alt text

**Review by danjacob-anders — APPROVED** [review]
All good!

**Pull request merged**